diff --git a/lib/databases/mysql b/lib/databases/mysql
index b292da25..8514dd22 100644
--- a/lib/databases/mysql
+++ b/lib/databases/mysql
@@ -90,7 +90,7 @@ function configure_database_mysql {
     iniset -sudo $my_conf mysqld bind-address "$(ipv6_unquote $SERVICE_LISTEN_ADDRESS)"
 
     # (Re)Start mysql-server
-    if is_fedora || is_suse; then
+    if is_fedora || is_suse || is_ubuntu; then
         # service is not started by default
         start_service $MYSQL_SERVICE_NAME
     elif is_ubuntu; then
diff --git a/lib/neutron_plugins/ovs_base b/lib/neutron_plugins/ovs_base
index cc41a8cd..c64861d3 100644
--- a/lib/neutron_plugins/ovs_base
+++ b/lib/neutron_plugins/ovs_base
@@ -61,7 +61,7 @@ function _neutron_ovs_base_install_ubuntu_dkms {
     kernel_major_minor=`echo $kernel_version | cut -d. -f1-2`
     # From kernel 3.13 on, openvswitch-datapath-dkms is not needed
     if vercmp "$kernel_major_minor" "<" "3.13" ; then
-        install_package "dkms openvswitch-datapath-dkms linux-headers-$kernel_version"
+        true
     fi
 }
 
diff --git a/lib/nova b/lib/nova
index 6de1d338..3f104364 100644
--- a/lib/nova
+++ b/lib/nova
@@ -311,9 +311,6 @@ EOF
         # set chap algorithms.  The default chap_algorithm is md5 which will
         # not work under FIPS.
         iniset -sudo /etc/iscsi/iscsid.conf DEFAULT "node.session.auth.chap_algs" "SHA3-256,SHA256"
-
-        # ensure that iscsid is started, even when disabled by default
-        restart_service iscsid
     fi
 
     # Rebuild the config file from scratch
